### ISSUE TITLE
Default Vite template not working because `"moduleResolution": "bundler"` is not supported

### ISSUE THREAD

**Issue by hexcowboy** [neoclide/coc-tsserver#429]
I imagine this is an issue with coc-tsserver's typescript version, since `"moduleResolution": "bundler"` was added in TS 5.x. I'm not sure how to update it

`system tsc version`
```
➜ tsc -v
Version 5.0.4
```

`coc-tsserver version`
```
- ✓ coc-tsserver Current version 2.1.3 is up to date.
```

![image](https://github.com/neoclide/coc-tsserver/assets/8162609/d0c726ba-f944-48a1-8bbd-fd451fc77516)

Reproduction:

Create a new Vite project with `react-swc-ts` template
```
pnpm create vite . --template react-swc-ts
```

**Comment by zaach** [comment]
A workaround for me was to run `:CocCommand tsserver.chooseVersion` and select the local version.

**Comment by eiji03aero** [comment]
It seems to be `typescript` package version has already been bumped up in latest master branch, but not released yet.
If they could release it, we should be able to go without creating local files for choosing version ...
When the maintainers got time, please make a new release 🙏 

<img width="1800" alt="Screen Shot 2023-09-16 at 5 40 31" src="https://github.com/neoclide/coc-tsserver/assets/13347338/b5fa9c1b-aecb-4e8f-841c-6b6c98cc4208">

**Comment by chemzqm** [comment]
Fixed on coc-tsserver@2.1.4
